test/index.test.js

```diff
@@ -327,6 +327,37 @@ describe('Feathers Objection Service', () => {
       })
     })
   })
+
+  describe('Transactions', () => {
+    let transaction
+
+    beforeEach(done => {
+      db.transaction(trx => {
+        transaction = {trx}
+        done()
+      }).catch(() => {})
+    })
+
+    it('works with commit', () => {
+      return people.create({name: 'Commit'}, {transaction}).then(() => {
+        return transaction.trx.commit().then(() => {
+          return people.find({ query: { name: 'Commit' } }).then((data) => {
```

Comment:
I think we would also want to check that the data isn't saved in the database before the commit.

Reply:
Ok, tried to add checks for this. There is a problem with that.
You need to have multiply connections to db for that.
And sqlite have problems with that: [1](https://knexjs.org/#Installation-pooling), [2](https://github.com/tgriesser/knex/blob/65509c38ef2d6c3878459aae69d8b31ac77e5cdf/src/dialects/sqlite3/index.js#L149).
Simple addition of `pool: { min: 2, max: 2 }` makes tests on my machine run `14s` against usual `4s` and sometimes some random test will fail with `"database is locked"`...

So it seems it does not worth the trouble, but just to be clear this one is passing with pool 2:
```js
it('works with commit', () => {
  return people.create({ name: 'Commit' }, { transaction }).then((data) => {
    return people.find({ query: { name: 'Commit' } }).then((data) => expect(data.length).to.equal(0)).then(() => {
      return transaction.trx.commit().then(() => {
        return people.find({ query: { name: 'Commit' } }).then((data) => expect(data.length).to.equal(1))
      })
    })
  })
})
```